### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#186] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-13 194707](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/ea626369-6ca7-4084-9a51-1bfa014896d4)

**Comment by cornflexxx** [comment, marked as answer]
$1)$ Per calcolare il limite di $P_{n,b,k}$  per  $b,n\to\infty$, ho utilizzato l'approssimazione di Stirling dei fattoriali di $b$ ed $n$. 
Per facilitarmi i conti ho applicato l'approssimazione al primo coefficiente binomiale , quindi $\binom{b}{k}$, con un po' di pazienza e alcune semplificazioni ho ottenuto questo risultato generale nel quale si possono sostituire gli altri termini : 
$$\binom{b}{k} \sim\frac{b^h}{(1-\frac{h}{b})^{b-h-\frac{1}{2}}}\frac{1}{h!}e^{-h}\sim\frac{b^h}{h!}\qquad\text{per b}\to\infty$$

Ho sostituito questa approssimazione all'interno di  $P_{n,b,k}$ , in questo modo raggruppiamo i fattoriali in "eccesso" in un ulteriore coefficiente binomiale $\binom{k}{h}$, quindi riscriviamo la formula nel seguente modo:
per $b,n\to\infty$
$$P_{n,b,k}=\binom{k}{h}\frac{b^hn^{k-h}}{(b+n)^k}$$ 
spezziamo la frazione in due:
$$\binom{k}{h}(\frac{b}{b+n})^h\cdot(\frac{n}{b+n})^{k-h}\qquad (i)$$ 

Sappiamo  che $\frac{b}{b+n}\to p\in(0,1)$ , ricordo che questo significa che la probabilità di estrarre una biglia di colore  bianco avviene con una probabilità $p$.La probabilità di estrarre una biglia nera invece, è proprio $1-p$, ed è definita proprio come $\frac{n}{b+n}$. Possiamo sotituire $p$ e $1-p$ in $(i)$:

$$\binom{k}{h}p^h\cdot (1-p)^{k-h}\qquad (ii)$$

$2)$ Dopo aver riscritto la formula  come $(ii)$, ci accorgiamo che la distribuzione della $v.a$ è diventata $\textbf{binomiale}$. Infatti la distribuzione binomiale rappresenta quei casi in cui si estraggono con rimpiazzo $k$ palline da un urna, contenente due tipi diversi di palline (indistinguibili o non numerate),  e si calcola la probabilità che esattamente $h$ palline siano del tipo prestabilito, sapendo che la probabilità di successo all' $i$-esima estrazione è $p$. In effetti la situazione descritta da $(ii)$ in altri termini è proprio questa: ponendo $b,n\to\infty$ , stiamo dicendo che l'estrazione $i$-esima è indipendente da ogni altra, in quanto abbiamo un insieme infinito sia rispetto a $B$ , sia rispetto ad $N$ (che rappresentano rispettivamente l'insieme delle palline bianche e nere).  Quindi se il numero di elementi da cui facciamo delle estrazioni senza rimpiazzo diventa molto più grande rispetto alle estrazioni che compiamo, allora possiamo ricondurre il comportamento della distribuzione ipergeometrica a quello di una distribuzione binomiale.

$$X\sim H(b,n,k) = X\sim B(k,p)\qquad\text{per $b,n\to\infty$} $$

(Se serve a qualcuno posso anche pubblicare la foto con i calcoli)
